NOTICE

```diff
@@ -281,6 +281,113 @@ Apache License Version 2.0:
       incurred by, or claims asserted against, such Contributor by reason
       of your accepting any such warranty or additional liability.
 
+=======================================================================
+Cephes's 3-Clause BSD License
+=======================================================================
+
+   Code derived from implementations in the Cephes Math Library should
+   mention its derivation and reference this note (ex. 'This function is
```

Comment:
This paragraph needs to be updated if the Cephes license is moved here, because this is no longer a note.

"and reference this note" -> "and reference the following Cephes 3-Clause BSD license. For example, 'This function is derived from the implementation of X in the Cephes Math Library. See the Cephes 3-Clause BSD license in PyTorch's NOTICE file.' The license is: